auth/issueToken.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require("jsonwebtoken");
+
+const jwtSecret = process.env.JWT_SECRET;
+
+const issueToken = async (user, res) => {
```

Comment:
Hmm, po pierwsze nie przekazujesz tu "res" bo niżej w login handler wywołujesz to jako return issueToken(user); czyli tylko z jednym parametrem i w takim wypadku będziesz chciała zwrócić zawartość tokenu a nie res.json

```
const issueToken = async (user) => {
  const payload = {
    id: user._id,
  };

  const token = jwt.sign(payload, jwtSecret);
  user.token = token;
  await user.save();
  return token;
};


```